jsapp/js/components/processing/sidebar/sidebarDisplaySettings.module.scss

```diff
@@ -0,0 +1,46 @@
+@use '~kobo-common/src/styles/colors';
+@use 'scss/sizes';
+
+.root {
+  display: flex;
+  justify-content: right;
+  padding-top: sizes.$x18;
+  padding-bottom: sizes.$x10;
+}
+
+.description {
+  padding-bottom: sizes.$x30;
+}
+
+.display {
+  padding-bottom: sizes.$x40;
+}
+
+.hasTranslation {
+  span::before {
+    content: '(';
+  }
+
+  span::after {
+    content: ')';
+  }
+}
+
+.options {
+  height: 100%;
+  padding-bottom: sizes.$x24;
+  overflow-y: scroll;
+}
+
+footer {
```

Comment:
If a CSS module file has tag selector, it is being applied globally to everything (to all `<footer>`s). CSS modules can only do localized styling with class selectors.